pulp_2to3_migration/tests/functional/test_file_migration.py

```diff
@@ -0,0 +1,99 @@
+import socket
+from time import sleep
+import unittest
+from pulpcore.client.pulpcore import (ApiClient as CoreApiClient, Configuration,
```

Comment:
Could we have every item on the new line? Similar to the style we have in pulpcore or pulp_rpm?

```
from ... import (
    module1,
    module2,
)
```

Reply:
yes! I will do that.